server/api/services/onboard.ts

```diff
@@ -82,6 +94,56 @@ function sendGitHubInvite(email, team, token) {
       })
       .then((response) => {
         if (response) resolve({ service: "GitHub", status: true });
+        else
+          reject({
+            service: "GitHub",
+            code: 500,
+            message: "Couldn't invite to GitHub",
+          });
+      });
+  });
+}
+
+function sendZohoInvite(email, role, profile, token) {
+  let body = {
+    users: [
+      {
+        role: role,
+        first_name: "Change",
```

Comment:
We can also create a list of unique names like ["John Doe", "Davy Jones", "Jane Doe"] and choose a random name from it.